.changeset/loud-starfishes-type.md

```diff
@@ -0,0 +1,7 @@
+---
+'@sap-cloud-sdk/generator': major
+---
+
+[Compatibility Note] Per default the OData generator does not change object names anymore to resolve conflicts.
+Instead, the generation process will fail.
+Switch on the `skipValidation` flag if you want to generate anyway with adjusted names.
```

Comment:
```suggestion
[Compatibility Note] The OData generator won't automatically rename identifiers to avoid name conflicts.
The generation process will fail if identifiers have conflicting names.
Switch on the `skipValidation` flag if you want to generate despite name conflicts and are okay with changed identifier names to avoid conflicts.
```

Reply:
hope this is better to understand